src/shipit_bot_uplift/shipit_bot_uplift/api.py

```diff
@@ -0,0 +1,113 @@
+import mohawk
+import requests
+import json
+import os
+from shipit_bot_uplift.helpers import ShipitJSONEncoder
+
+
+class ApiClient(object):
```

Comment:
Not in this PR but in general it would be nice if we would be able to generate this from ``api.yml`` at least partially and have a common way to access all of the APIs